### PR TITLE
Removed AX Timeout

### PR DESCRIPTION
**Pull request by ghost**
## Problem
The AX Timeout and the fact that MJPEG Screenshots are taken on the main thread, were causing Appium Sessions to fail on real devices. 
Steps to reproduce:
1. Start WDA
2. Forward local port to 9100 on device
3. Start MJPEG Stream using nc localhost localport
4. Wait for at least one Screenshot to be taken. 
4. Starting any Appium test will deadlock even if MJPEG is inactive again.
## Solution
Remove AX Timeout.

### CODE REVIEW

File: WebDriverAgentLib/Utilities/FBMjpegServer.m [hunk 1]
```diff
@@ -82,16 +82,14 @@ - (void)streamScreenshot
   CGFloat compressionQuality = FBConfiguration.mjpegServerScreenshotQuality / 100.0f;
   id<XCTestManager_ManagerInterface> proxy = [FBXCTestDaemonsProxy testRunnerProxy];
   dispatch_semaphore_t sem = dispatch_semaphore_create(0);
-  [proxy _XCT_setAXTimeout:SCREENSHOT_TIMEOUT reply:^(int res) {
```
Comment (mykola-mokhnach) [inline WebDriverAgentLib/Utilities/FBMjpegServer.m:85]: should we also remove the unused SCREENSHOT_TIMEOUT constant?

Comment (ghost) [reply WebDriverAgentLib/Utilities/FBMjpegServer.m:85]: it is still used in the semaphore timeout, right?

Comment (mykola-mokhnach) [reply WebDriverAgentLib/Utilities/FBMjpegServer.m:85]: yep, makes sense

**Comment by mykola-mokhnach** [comment]
Do you mind signing the CLA?

**Comment by ghost** [comment]
I actually have done it. is there a way to make the bot check again?

**Comment by mykola-mokhnach** [comment]
You should check the email, which is set for your local git account. It must be the same as in the CLA

**Comment by ghost** [comment]
maybe because i had the "keep my email private" setting enabled in github. i disabled it now

**Pull request merged**